crawler/src/main/java/nl/fhict/happynews/crawler/crawler/QuoteCrawler.java

```diff
@@ -70,6 +70,7 @@ void crawl() {
         for (Quote quote : entity.getContents().getQuotes()) {
             Post toAdd = new Post();
             toAdd.setSource("Inspirational Quote");
+            toAdd.setSourceName("quote");
```

Comment:
Yeah, we're gonna have to make this consistent throughout the application at some point in the future.